doc/sphinx/language/core/records.rst

```diff
@@ -235,34 +235,12 @@ the errors of inductive definitions, as described in Section
 Primitive Projections
 ~~~~~~~~~~~~~~~~~~~~~
 
-.. flag:: Primitive Projections
-
-   This :term:`flag` turns on the use of primitive
-   projections when defining subsequent records (even through the ``Inductive``
```

Comment:
Maybe keeping the mention that this applies even with the records defined with the `Inductive` or `CoInductive` commands would be useful (actually, it is kept in the description of the `flag`...). By the way, you forgot to list the attribute in the supported attributes for the `Inductive` command.

Reply:
I didn't mention it for Inductive since it is technically not supported. I believe only for the `:= { .. }` syntax.

Reply:
Alright, I've modified all the other mentions of the attribute to specify that it is valid only when using record syntax. This includes `Inductive`.